broker-daemon/broker-rpc/broker-rpc-server.js

```diff
@@ -79,21 +52,24 @@ class BrokerRPCServer {
     this.pubKeyPath = pubKeyPath
     this.privKeyPath = privKeyPath
     this.disableAuth = disableAuth
+    this.rpcUser = rpcUser
+    this.rpcPass = rpcPass
+    this.auth = basicAuth.verify.bind(this)
 
     this.protoPath = path.resolve(BROKER_PROTO_PATH)
 
     this.server = new grpc.Server()
 
-    this.adminService = new AdminService(this.protoPath, { logger, relayer, engines })
+    this.adminService = new AdminService(this.protoPath, { logger, relayer, engines, basicAuth: this.auth })
```

Comment:
I don't think the property should be named `basicAuth`, it's leaking an implementation detail into places it doesn't need to be. Any reason it can't be just `auth` ?